doc/introduction/chemistry.rst

```diff
@@ -86,66 +86,63 @@ coordinates of each atomic species.
 Solving the Hartree-Fock equations
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The :func:`~.meanfield_data` function uses the `OpenFermion-PySCF <https://github
+The :func:`~.meanfield` function uses the `OpenFermion-PySCF <https://github
 .com/quantumlib/OpenFermion-PySCF>`_ and `OpenFermion-Psi4 <https://github
 .com/quantumlib/OpenFermion-Psi4>`_ plugins to solve the Hartree-Fock equations for the molecule
 using the electronic structure packages `PySCF <https://github.com/sunqm/pyscf>`_ and `Psi4
 <http://psicode.org/>`_, respectively.
 
-For this, it is required to specify a string to label the molecule, its net charge, the `spin
-multiplicity <https://en.wikipedia.org/wiki/Multiplicity_(chemistry)>`_ and the  `atomic basis
-functions <https://www.basissetexchange.org/>`_.
+For this, it is required to specify a string to label the molecule. Furthermore, the net charge,
+the `spin multiplicity <https://en.wikipedia.org/wiki/Multiplicity_(chemistry)>`_ and the  `atomic basis functions <https://www.basissetexchange.org/>`_ can also be specified.
 
 .. code-block:: python
 
     geometry = qml.qchem.read_structure('h2o.SDF')
-    hf_data = qml.qchem.meanfield_data(
+    hf_file = qml.qchem.meanfield(
         'water',
         geometry,
         charge=0,
-        multiplicity=1,
-        basis_set='sto-3g',
-        qc_package='pyscf'
+        mult=1,
+        basis='sto-3g',
+        package='pyscf'
     )
 
-The output variable ``hf_data`` stores the path to the directory containing the file ``'water.hd5'``
-with the Hartree-Fock electronic structure of the water molecule.
+The output variable ``hf_file`` stores the full path to the file ``'water.hd5'`` containing
+the Hartree-Fock electronic structure of the water molecule.
 
 Mapping the Hamiltonian to the Pauli basis
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The function :func:`~.active_space` is used to create an `active space <https://en.wikipedia
-.org/wiki/Complete_active_space>`__  by classifying the Hartree-Fock molecular orbitals as doubly-occupied,
-active, and external orbitals. Within this approximation, a certain number of *active electrons*
-can populate the *active orbitals*.
+.org/wiki/Complete_active_space>`__  by classifying the Hartree-Fock molecular orbitals as
+core, active, and external orbitals. Within this approximation, a certain number of *active electrons* can populate the *active orbitals*.
 
 .. code-block:: python
 
-    d_occ_indices, active_indices = qml.qchem.active_space(
-        'water',
-        hf_data,
-        n_active_electrons=2,
-        n_active_orbitals=2
+    water = MolecularData(filename=hf_file)
+    core, active = qml.qchem.active_space(
+        water.n_electrons,
+        water.n_orbitals,
```

Comment:
I would prefer to rename `n_electrons` and `n_orbitals` to just `electrons` and `orbitals`

Reply:
My comment above regarding the "n" prefix also applies here.

Reply:
Indeed, here we don't have this flexibility as `n_electrons` and `n_orbitals` are attributes of `water` whose type is `MolecularData`, the OpenFermion class.